lib/Transforms/Scalar/RewriteStatepointsForGC.cpp

```diff
@@ -69,7 +69,7 @@ namespace {
 struct RewriteStatepointsForGC : public FunctionPass {
   static char ID; // Pass identification, replacement for typeid
 
-  RewriteStatepointsForGC() : FunctionPass(ID) {
+  RewriteStatepointsForGC() : FunctionPass(ID), MustTrackBasePointers(true) {
```

Comment:
I'm fine with initializing it to true, given that's the default behavior except for CLR.